src/central.ts

```diff
@@ -0,0 +1,78 @@
+import { NativeEventEmitter, NativeModules } from 'react-native'
+import { BaseBLE } from './base'
+import { startCentral, write, scan, connect } from './functions'
+
+export class Central extends BaseBLE {
+  async sendMessage(message: string) {
+    return await write(message)
+  }
+  async start(
```

Comment:
not 100% sure I can follow you on this one😅

Reply:
```ts
// BADDD GRRRRR
const a = (a:string,b:string, c:string, d:string, e:string) => console.log(a,b,c,d,e)
// GOOD :)))))))
const b  = (options:{a:string, b:string, c:string, d:string}) => console.log(options)
```

Reply:
hmmm so I made this an options: Object thing but since the sdk's method interface is different and has different names for the args I have to explicitly pass them to sdk.start. Not sure if you had something else in mind